src/etc/inc/openvpn.inc

```diff
@@ -447,6 +447,16 @@ function openvpn_get_digestlist() {
 	return $digests;
 }
 
+function openvpn_get_keydirlist() {
+	$keydirs = array(
+		'-1' => gettext('Use default keydir direction'),
+		'2' => gettext('No keydir direction'),
```

Comment:
The description for this setting should be "Both Directions". This option should also be last, not one of the first options.